reflex-dom-core/src/Reflex/Dom/Builder/Static.hs

```diff
@@ -279,7 +293,7 @@ instance SupportsStaticDomBuilder t m => DomBuilder t (StaticDomBuilderT t m) wh
       Just setContents -> hold (escape initialContents) $ fmapCheap escape setContents --Only because it doesn't get optimized when profiling is on
     return $ CommentNode ()
   {-# INLINABLE element #-}
-  element elementTag cfg child = do
+  element elementTag cfg child = prependCallStackComment callStack $ do
```

Comment:
Probably want to do this for `inputElement` / `selectElement` / `textAreaElement` as well